proxy/proxy.go

```diff
@@ -31,41 +31,93 @@ const (
 	proxyBufferSize = 8192
 	proxyErrorFmt   = "proxy: %s"
 
-	// TODO: this should be fine tuned, yet, with benchmarks.
-	// In case it doesn't make a big difference, then a lower value
-	// can be safer, but the default 2 turned out to be too low during
-	// previous benchmarks.
-	//
-	// Also: it should be a parameter.
-	idleConnsPerHost = 64
+	// The default value set for http.Transport.MaxIdleConnsPerHost.
+	DefaultIdleConnsPerHost = 64
+
+	// The default period at which the idle connections are forcibly
+	// closed.
+	DefaultCloseIdleConnsPeriod = 12 * time.Second
```

Comment:
Is this default also copied from somewhere else or are we introducing it? If we're introducing it, I'd increase this to 20secs/30secs to align with the default DNS TTL (20sec) or the JVMs 30sec